go.mod

```diff
@@ -20,7 +20,7 @@ require (
 	github.com/decred/dcrd/dcrec/edwards v1.0.0
 	github.com/dustin/go-humanize v1.0.1
 	github.com/dymensionxyz/gerr-cosmos v1.0.0
-	github.com/dymensionxyz/sdk-utils v0.2.7
+	github.com/dymensionxyz/sdk-utils v0.2.8-0.20240821082233-d3adcc68e4e1
```

Comment:
utACK, please include the new pretty event from `sdk-utils`

Reply:
yeah, just pushed it